include/rcutils/error_handling.h

```diff
@@ -39,14 +39,16 @@ extern "C"
 #include "rcutils/types/rcutils_ret.h"
 #include "rcutils/visibility_control.h"
 
-#ifdef __STDC_LIB_EXT1__
+#ifdef defined(__STDC_LIB_EXT1__) && !defined(RCUTILS_NO_FILESYSTEM)
```

Comment:
```suggestion
#if defined(__STDC_LIB_EXT1__) && !defined(RCUTILS_NO_FILESYSTEM)
```

Reply:
error_handling.h has been substituted by error_handling.h.in